src/host/libpixyusb_swig/helper_commands.cpp

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <signal.h>
+#include <string.h>
+#include "pixy.h"
+
+
+// frame should be at least 64000 bytes
+int pixy_cam_get_frame(uint8_t mode, uint16_t xoffset, uint16_t yoffset, uint16_t width, uint16_t height, uint8_t *frame)
```

Comment:
what happens if the frame is NOT at least 64k?
Should we add a check for that minimum size?

Reply:
Since it's a pointer to a buffer there really is no way for this function to truly check this. I have to see if there is anything I can do it the swig/python wrapper code. No guarantees. I won't spend too much time because all of this will be gone when we find a better camera platform.